chia/wallet/wallet_state_manager.py

```diff
@@ -1631,10 +1631,6 @@ async def reorg_rollback(self, height: int) -> List[uint32]:
                 remove: bool = await wallet.rewind(height)
                 if remove:
                     remove_ids.append(wallet_id)
-            if wallet.type() == WalletType.NFT.value:
-                assert isinstance(wallet, NFTWallet)
-                if await wallet.get_nft_count() == 0:
-                    remove_ids.append(wallet_id)
```

Comment:
it looks like the test covers this change, but is there a test covering the change in `wallet_rpc_api.py` too?

Reply:
For DID, tests/wallet/did_wallet/test_did.py#test_get_info
For NFT, tests/wallet/nft_wallet/test_nft_wallet.py#test_nft_set_did

Reply:
but those tests passed before the fix, right?

Reply:
Covered in the unit test